### DISCUSSION TITLE
Cookie banner as relative content

### DISCUSSION THREAD

**Discussion by karkkant** [orestbida/cookieconsent#599] [General]
I'm trying to place the cookie banner as relative content on top of the page before anything else. It would be non-floating (non intrusive) bar. I think it's possible to override all those "fixed" positions but my issue is that the script always loads DOM content at the end of body.

Is there any way to create this kind of layout currently?
If not, could there be a feature/property where you could define a placeholder element where the script loads the banner?

Reasoning why this kind of layout option would be great is because these floating panels hide elements and create accessibility issues.

**Comment by orestbida** [comment]
You can append the generated html markup inside an html element via the [`root`](https://cookieconsent.orestbida.com/reference/configuration-reference.html#root) option.
